types.go

```diff
@@ -35,3 +36,8 @@ type BJJSignatureProof struct {
 	IssuerAuthClaimMTP    *merkletree.Proof
 	IssuerAuthNonRevProof ClaimNonRevStatus // IssuerAuthClaim non revocation proof
 }
+
+type NullifierInputs struct {
+	Salt      *big.Int
+	Nullifier *big.Int
```

Comment:
I would not pass the nullifier to the circuit, it should be output from the circuit.

Reply:
Fixed as discussed